primefaces/src/main/java/org/primefaces/model/charts/axes/AxesScale.java

```diff
@@ -34,6 +34,8 @@
 
     private boolean display = true;
     private Number weight;
+    private Number suggestedMax;
+    private Number suggestedMin;
```

Comment:
Please see my 2nd comment in ChartRenderer.java.

Since these options are 'per axis', there is no way to configure them for either x or y axis here.